stubs/urllib3/urllib3/fields.pyi

```diff
@@ -1,13 +1,31 @@
-from typing import Any
+from collections.abc import Callable, Mapping
+from typing import Any, Union
+from typing_extensions import TypeAlias
 
-def guess_content_type(filename, default=...): ...
-def format_header_param(name, value): ...
+_FieldValue: TypeAlias = str | bytes
+_FieldValueTuple: TypeAlias = Union[_FieldValue, tuple[str, _FieldValue], tuple[str, _FieldValue, str]]
+
+def guess_content_type(filename: str | None, default: str = ...) -> str: ...
+def format_header_param_rfc2231(name: str, value: _FieldValue) -> str: ...
+def format_header_param_html5(name: str, value: _FieldValue) -> str: ...
+def format_header_param(name: str, value: _FieldValue) -> str: ...
```

Comment:
To match the implemenation:

```suggestion
format_header_param = format_header_param_html5
```

Reply:
Good catch. I had read 2.x.x code when it was supposed to be https://github.com/urllib3/urllib3/blob/1.26.x/src/urllib3/fields.py where it is required